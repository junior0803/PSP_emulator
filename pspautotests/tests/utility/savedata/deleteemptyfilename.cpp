#include "emptyfilename.h"

extern "C" int main(int argc, char **argv) {
	initDisplay();

	runEmptyFileNameTrials("auto delete:", SCE_UTILITY_SAVEDATA_TYPE_AUTODELETE);
	runEmptyFileNameTrials("delete:", SCE_UTILITY_SAVEDATA_TYPE_DELETE);
	runEmptyFileNameTrials("list delete:", SCE_UTILITY_SAVEDATA_TYPE_LISTDELETE);
	runEmptyFileNameTrials("list all delete:", SCE_UTILITY_SAVEDATA_TYPE_LISTALLDELETE);
	runEmptyFileNameTrials("delete data:", SCE_UTILITY_SAVEDATA_TYPE_DELETEDATA);
	runEmptyFileNameTrials("erase secure:", SCE_UTILITY_SAVEDATA_TYPE_ERASESECURE);
	runEmptyFileNameTrials("erase:", SCE_UTILITY_SAVEDATA_TYPE_ERASE);

	return 0;
}