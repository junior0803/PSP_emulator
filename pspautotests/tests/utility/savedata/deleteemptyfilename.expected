[x] auto delete:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:9): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] loaddata:Orignal data

[x] delete:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:10): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] loaddata:Orignal data

[x] list delete:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:6): 00000000
[r] result: 00000000 => 80110347
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] loaddata:Orignal data

[x] list all delete:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:7): 00000000
[r] result: 00000000 => 00000001
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] loaddata:Orignal data

[x] delete data:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:21): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] loaddata:Orignal data

[x] erase secure:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:19): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] erase:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:20): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data
